app/main.py

```diff
@@ -1,13 +1,16 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"], "name": error["code"],
+            "source": "flake8"}
```

Comment:
```suggestion
    return {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"], "name": error["code"],
        "source": "flake8"
    }
```
It clearer formatting and common in the Python community